letsencrypt-win-simple/Scripts/ImportExchange.ps1

```diff
@@ -1,56 +1,95 @@
 param(
```

Comment:
Could we add comment-based help with a `.SYNOPSIS`,  and an `.EXAMPLE` in the comment-based help how to call this using Let's encrypt? If there are more than one option to call we should add more than one `.EXAMPLE`.

Reply:
I'm a little torn on whether to put it there or just in the wiki. I've never done that so I can see how difficult it is. probably cleaner than the way i have remarks currently

Reply:
Better? That's pretty cool. 
![image](https://user-images.githubusercontent.com/15981748/34961097-6c596836-fa03-11e7-9843-8f8028ca1ea6.png)